docs/IDE-INTEGRATION.md

```
@@ -236,10 +236,10 @@ npx tsx /path/to/claude-octopus/mcp-server/src/index.ts
 
 ### Provider not available
 
-If Codex or Gemini workflows fail, check:
+If Codex or Antigravity workflows fail, check:
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)